### DISCUSSION TITLE
How to enable output of 75th Percentile metric?

### DISCUSSION THREAD

**Discussion by dan-newscorp** [artilleryio/artillery#2620] [Artillery]
Hi,
Am loving using Artillery and Artillery + Playwright!

As our focus is on Core Web Vitals, and the target thresholds are in the 75th percentile... how do we enable outputting that quantile?

I've seen some random screenshots in issues etc where it outputs it, but currently unable to find a solution.

Thanks in advance,
Cheers,
Dan

![Screenshot 2024-03-27 at 11 20 52 am](https://github.com/artilleryio/artillery/assets/68039408/05dac577-c911-4608-a684-78bfb3a8ff71)

**Comment by hassy** [comment]
Hi @dan-newscorp 👋 There's no way to enable p75 reporting in the console output right now, but we'll add that. In the meantime you can get p75 values for all metrics from a JSON report produced by the [`--output` flag](https://www.artillery.io/docs/reference/cli/run#options).

**Reply by dan-newscorp** [reply]
Hi @hassy 
Ok no worries, thats great its going to be added though. 
That output flag will be great for now.
Thanks for your reply!
